src/stringify.js

```diff
@@ -35,7 +35,10 @@ function stringify(arr, offset = 0) {
     byteToHex[arr[offset + 14]] +
     byteToHex[arr[offset + 15]]
   ).toLowerCase();
+}
 
+export function stringify(arr, offset = 0) {
```

Comment:
Having multiple ways of importing the same method is confusing. 

```suggestion
function stringify(arr, offset = 0) {
```

Reply:
Personally, I think default exports are very unfriendly to auto-import workflows when developing code. But so be it.